business/services/ConfigurationService.go

```diff
@@ -0,0 +1,40 @@
+package services
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/decentralized-cloud/Tenant/business/contracts"
+)
+
+// ConfigurationService implements the service that provides environment variables
+type ConfigurationService struct {
+}
+
+// NewConfigurationService creates new instance of the ConfigurationService, setting up all dependencies and returns the instance
+// Returns the new service or error if something goes wrong
+func NewConfigurationService(contracts.ConfigurationServiceContract, error) {
```

Comment:
I think you have the argument inside out, aren't they supposed to be the return arguments?

Reply:
not sure if fix applied is correct.